cdap-explore-jdbc/src/main/java/co/cask/cdap/explore/jdbc/ExploreStatement.java

```diff
@@ -43,7 +43,7 @@
 public class ExploreStatement implements Statement {
   private static final Logger LOG = LoggerFactory.getLogger(ExploreStatement.class);
 
-  private int fetchSize = 50;
+  private int fetchSize = 1000;
```

Comment:
this variable is not a default, but the actual, mutable fetch size for the statement. java.sql.Statement defines a setFetchSize() for people to set this.

50 is the Hive jdbc driver's default, which I guess is why we have this at 50.  But it's very slow when working with any decent number of rows.  It's also more likely to overload the cdap router.